players/base.py

```diff
@@ -40,7 +40,10 @@ def __init__(self, name: str, id: int, battle_menu: List[int]):
         self.struct_offset = id * PlayerMagicNumbers.CHAR_STRUCT_SIZE
         self.char_rng = 20 + id
         self.battle_menu = battle_menu
-
+    
+    def actor_id(self):
```

Comment:
I would rather see this as a `property`:

```python
@property
def actor_id(self):
    return self.id + 1
```

Then we can do `Auron.actor_id` (no `()`). It's a much more pythonic way of doing something like this.

Reply:
That's new and fun and I didn't know existed. :D

Reply:
Updated, tested, and working.